server/form-pages/placement-application/request-a-placement/reasonForPlacement.ts

```diff
@@ -25,7 +25,17 @@ export default class ReasonForPlacement implements TasklistPage {
   }
 
   next() {
-    return this.body.reason === 'rotl' ? 'previous-rotl-placement' : 'additional-placement-details'
+    if (this.body.reason === 'rotl') {
+      return 'previous-rotl-placement'
+    }
+    if (this.body.reason === 'additional_placement') {
+      return 'additional-placement-details'
+    }
+    if (this.body.reason === 'release_following_decision') {
+      return 'decision-to-release'
+    }
+
+    return ''
```

Comment:
Wonder if we should have a case statement and throw an error as the default here? The reason should be one of those three reasons, otherwise we have a problem

Reply:
Yeah good point